package/utils/util-linux/Makefile

```diff
@@ -172,7 +172,7 @@ endef
 define Package/fdisk
 $(call Package/util-linux/Default)
   TITLE:=manipulate disk partition table
-  DEPENDS:= +libblkid +libsmartcols +libfdisk
+  DEPENDS:= +libblkid +libsmartcols +libfdisk +libncursesw
```

Comment:
Does it also build with just `libncurses`?  Why does it require the wide version?  Is NLS turned on?